packages/asc-ui/src/components/List/ListStyle.ts

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 import styled from '@datapunt/asc-core'
 import { color } from '../../utils'
 
-export interface Props extends React.HTMLAttributes<HTMLUListElement> {}
+export interface ListStyleProps
+  extends React.HTMLAttributes<HTMLUListElement> {}
 
-export default styled.ul<Props>`
+export default styled.ul<ListStyleProps>`
```

Comment:
Don'we export them as props everywhere and handle the naming in the component that imports it?

Reply:
sure! thanks!